test/client/index.js

```diff
@@ -342,7 +348,7 @@ Client.prototype.destroyAccount = function () {
 }
 
 Client.prototype.forgotPassword = function (lang) {
-  this._clear()
```

Comment:
If I remember correctly, this was removed because it was unnecessarily clearing the test client's data.